zerver/tests/test_signup.py

```diff
@@ -843,6 +854,29 @@ def test_successful_invite_user_as_guest_from_admin_account(self) -> None:
         self.assertFalse(invitee_profile.is_realm_admin)
         self.assertTrue(invitee_profile.is_guest)
 
+    def test_successful_invite_with_support_param(self) -> None:
+        invitor = self.example_user('iago')
+        invitor.is_staff = True
```

Comment:
This does not do anything since you are not saving the object. Just add an assert statment here instead that checks that invitor aka iago is staff.